app/src/test/java/ch/hikemate/app/model/facilities/FacilitiesRepositoryOverpassTest.kt

```diff
@@ -63,6 +63,18 @@ class FacilitiesRepositoryOverpassTest {
           "amenity": "toilets"
         }
       }"""
+
+    private const val VALID_RESPONSE_ELEMENT_WITH_MULTIPLE_TAGS =
+        """
+      {
+        "type": "node",
+        "id": 313827518,
+        "lat": 50.9620,
+        "lon": 5.12740,
+        "tags": {
+          "amenity": "parking"
+        }
+      }"""
```

Comment:
Nitpick: This doesn't look like the val has multiple tags, could you improve the variable name please?

Reply:
Oops